docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -86,14 +106,16 @@ You can set an alert's status to indicate whether it needs to be investigated
 
 To change an alert's status, do one of the following:
 
-* In the alert's row, click the *More actions* menu (*...*) in the Alerts table, then select the appropriate status (*Mark as open*, *Mark as acknowledged*, or *Mark as closed*).
-* In the Alerts table, select all the alerts you want to change, click on the drop-down at the upper-left above the table, and then select *Mark as open*, *Mark as acknowledged*, or *Mark as closed*.
+* In the alert's row, click the *More actions* menu (*...*) in the Alerts table, then select a status.
+
+* In the Alerts table, select all the alerts you want to change, click on the drop-down at the upper-left above the table, and then select a status.
```

Comment:
```suggestion
* In the Alerts table, select all the alerts you want to change, click *Selected _N_ alerts* at the upper-left above the table, and then select a status.
```
Not sure if X or N would be better, and the picture makes it pretty clear as-is, but I think naming the button couldn't hurt

Reply:
Good call, lowercase _x_ is the [preferred format](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#general-guidelines-for-writing-about-the-ui):
```suggestion
* In the Alerts table, select all the alerts you want to change, click *Selected _x_ alerts* at the upper-left above the table, and then select a status.
```